### PR TITLE
Add scraping functionality for Apple

### LINKED ISSUE #72: [Backend] Scrape Apple
## Description
We want to be able to successfully *and* consistently scrape job postings from Apple's Careers site
## Tasks
- [ ] Scrape job postings From Apple's Careers Site
- [ ] Parse data from Apple job post

## To Discuss
Same as #68

### PR DESCRIPTION
**Pull request by lilyxmeng**
## Description
Fixes #72 
#### What is the purpose of your change?
To add code to scrape the job titles, links, and locations of Apple internships off the Apple website.

#### What areas of the application does this change impact? (changes API, folder structure, etc)
scraper/main.py

#### What features/fixes are deferred/not handled in this PR?
N/A, meets what is required for ticket #72.

## Checklist
- [ ] As stated in ticket #72 discussion section, discuss and determine how we want to organize the current data into the database.

### CODE REVIEW

File: scraper/main.py [hunk 1]
```diff
@@ -1,2 +1,47 @@
-if __name__ == "__main__":
-    print("Hello World")
+import requests
+from bs4 import BeautifulSoup
+
+# URLs for FAANG internship listings
+urls = {
+    'Facebook': 'https://www.metacareers.com/jobs/?is_leadership=0&is_in_page=0',
+    'Amazon': 'https://www.amazon.jobs/en/search?base_query=internship',
+    'Apple': 'https://jobs.apple.com/en-ca/search?team=internships-STDNT-INTRN',
```
Comment (jwu910) [inline scraper/main.py:8]: https://github.com/BobaTalks/internship-tracker/pull/104#pullrequestreview-1407023905

Same comment on this url

**Review by jwu910 — COMMENTED** [review]
What's the best way to go about testing this?

File: scraper/main.py (hunk 1 above)
Comment (lilyxmeng) [reply scraper/main.py:8]: Sorry, the link should be https://jobs.apple.com/en-ca/search?location=canada-CANC+united-states-USA+mexico-MEXC&team=internships-STDNT-INTRN. I'm not sure how to change it, should I make another pull request?

Comment (jwu910) [reply scraper/main.py:8]: @lilyxmeng with pull requests on GitHub, you should be set up so your branch is tracked to this pull request. So you can make the change and commit to your repository and the pull request will update.

**Review by a-camarillo — COMMENTED** [review]
Looks good! If you can just make sure to lint with Black so all tests are passing then I think we're good to merge this.

If you haven't used black before just make sure you have it installed in your virtual environment and then in the scraper directory you can just run `black .` and this should format files if needed

**Comment by jwu910** [comment]
@lilyxmeng can we support you in any way with this?

**Comment by lilyxmeng** [comment]
> @lilyxmeng can we support you in any way with this?

I think this code should be good to merge. I'm not aware of any other issues this code has that I need to address.

**Comment by jwu910** [comment]
@lilyxmeng the checks aren't passing, @a-camarillo left a [comment](https://github.com/BobaTalks/internship-tracker/pull/111#pullrequestreview-1415067384) last week about the code format. Can you address this first?

**Comment by lilyxmeng** [comment]
> @lilyxmeng the checks aren't passing, @a-camarillo left a [comment](https://github.com/BobaTalks/internship-tracker/pull/111#pullrequestreview-1415067384) last week about the code format. Can you address this first?

It should be good to merge now.

**Review by michachan — APPROVED** [review]

**Pull request merged**